sentry/src/main/java/io/sentry/SentryMeasurementUnit.java

```diff
@@ -0,0 +1,37 @@
+package io.sentry;
+
+import java.util.Locale;
+import org.jetbrains.annotations.NotNull;
+
+public enum SentryMeasurementUnit {
```

Comment:
The value class does not have a `Sentry` prefix.

Reply:
Do you mean we should remove it here?

Reply:
Yes, the other one is called `MeasurementValue`.

Reply:
How about renaming it to `SentryMeasurementValue` rather? I feel like `MeasurementUnit` is a pretty common name, so I think it's better to keep it with the Sentry prefix for distinction

Reply:
I'd leave this with you both, `MeasurementValue` is `@ApiStatus.Internal` so I'd be fine with it, btw, remember to remove such annotations now since they are public APIs.

Reply:
`MeasurementValue` is not public. You only call `setMeasurement` on the span but don't know that it will be converted internally. So I'd keep it marked as internal. Also feel like `MeasurementValue` is likely to have conflicts.

Reply:
as agreed with @adinauer we'll keep it as is